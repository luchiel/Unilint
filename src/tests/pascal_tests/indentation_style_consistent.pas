var
    a_global: integer;
    
procedure increase_a;
begin
  a_global := a_global + 1;
end;

var
    i, j: integer;
    begin
    i := 1;
    j := 6;
    while i < 10 do begin
        if i > j then
          Inc(i)
        else
            i := i + 2;
    end;
end.
