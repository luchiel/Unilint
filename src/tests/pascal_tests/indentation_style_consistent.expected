6:1: indent too small
11:1: indent too deep
16:1: indent too small
